backend/src/ml_space_lambda/data_access_objects/project_group.py

```diff
@@ -0,0 +1,125 @@
+#
+#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+# Project Group Table Data Access Object
+from __future__ import annotations
+
+import json
+from typing import List, Optional
+
+from dynamodb_json import json_util as dynamodb_json
+
+from ml_space_lambda.data_access_objects.dynamo_data_store import DynamoDBObjectStore
+from ml_space_lambda.enums import EnvVariable, Permission
+from ml_space_lambda.utils.common_functions import serialize_permissions
+from ml_space_lambda.utils.mlspace_config import get_environment_variables
+
+
+class ProjectGroupModel:
+    def __init__(
+        self,
+        group_name: str,
+        project_name: str,
+        permissions: Optional[List[Permission]] = None,
+    ):
+        permissions = permissions if permissions else []
+        self.group_name = group_name
+        self.project = project_name
+        self.permissions = permissions
```

Comment:
can't we combine line 38 and this for something like:
`self.permissions = permissions if permissions else []`

Reply:
Better yet I think I'll just default the parameter to `[]` so if `None` is passed in we get the desired effect.